smart_campus/app/templates/app/index.html

```diff
@@ -1,4 +1,11 @@
-{% extends "app/base.html" %}
+{% extends 'app/base.html' %}
 {% load static %}
-
-{% block title %}Smart Campus{% endblock %}
+{% block stylesheet %}
+  {{ block.super }}
+  <link rel="stylesheet" type="text/css" href="{% static "app/css/index.css" %}">
+{% endblock %}
```

Comment:
It would be much better if you unify your blank line style between blocks in django template.
I suggest that you can add one between blocks.

Reply:
okay, it seems like my indentation setting is broken, I will check about it.
BTW, fixed it.